### PR TITLE
Second level initial code

### PR DESCRIPTION
**Pull request by protito**
If you want to check the second level uncomment line 108 and comment 107 in AppDelegate.cpp

### CODE REVIEW

File: Classes/UranusScene.cpp [hunk 1]
```diff
@@ -0,0 +1,105 @@
+//
+//  UranusScene.cpp
+//  akua
```
Comment (jogrimst) [inline Classes/UranusScene.cpp]: "akua" --> "nya"

File: RAW/Nya spine/Untitled1.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
+{"skeleton":{"images":"images/"},
```
Comment (jogrimst) [inline RAW/Nya spine/Untitled1.json]: Should the name of this file be `Untitled1.json`?

File: RAW/Nya spine/Untitled1.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+"bones":[{"name":"root"}],
+"slots":[
+	{"name":"pierna2","bone":"root","attachment":"pierna2"},
```
Comment (jogrimst) [inline RAW/Nya spine/Untitled1.json]: Rename body parts from Spanish to English.

File: RAW/bitacora.tps [hunk 4]
```diff
@@ -0,0 +1,293 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<data version="1.0">
+    <struct type="Settings">
+        <key>fileFormatVersion</key>
+        <int>4</int>
+        <key>texturePackerVersion</key>
+        <string>4.4.0</string>
+        <key>fileName</key>
+        <string>D:/Developer/Games/Nya/RAW/bitacora.tps</string>
```
Comment (jogrimst) [inline RAW/bitacora.tps]: Is this file supposed to be included for versioning control?

File: RAW/loading.tps [hunk 5]
```diff
@@ -0,0 +1,277 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<data version="1.0">
+    <struct type="Settings">
+        <key>fileFormatVersion</key>
+        <int>4</int>
+        <key>texturePackerVersion</key>
+        <string>4.4.0</string>
+        <key>fileName</key>
+        <string>D:/Developer/Games/Nya/RAW/loading.tps</string>
```
Comment (jogrimst) [inline RAW/loading.tps]: It seems like this is a developer-specific path. Should the file (or folder) be added to `.gitignore`?

File: Classes/UranusScene.cpp (hunk 1 above)
Comment (protito) [inline Classes/UranusScene.cpp]: done

File: RAW/Nya spine/Untitled1.json (hunk 2 above)
Comment (protito) [inline RAW/Nya spine/Untitled1.json]: Renamed the file and changed the names to English

File: RAW/Nya spine/Untitled1.json (hunk 3 above)
Comment (protito) [inline RAW/Nya spine/Untitled1.json]: Removed the file an added the generating script

File: RAW/bitacora.tps (hunk 4 above)
Comment (protito) [inline RAW/bitacora.tps]: This is the way the files are saved but it actually uses relative paths, https://www.codeandweb.com/texturepacker/documentation
See Data file section

File: RAW/loading.tps (hunk 5 above)
Comment (protito) [inline RAW/loading.tps]: Same as above

**Pull request merged**